Header header #for timestamp

ComponentReferent[] status # an array of components being reported on